salt/states/saltmod.py

```diff
@@ -371,6 +379,7 @@ def function(
                 ).format(fun, tgt, str(False))
         ret['result'] = None
         return ret
+    cmd_ret = __salt__['saltutil.cmd'](tgt, fun, **cmd_kw)
```

Comment:
That's my fault, I added the extra one outside of the try block to see a stack trace of an exception, that extra one can be removed.

Sent from my iPhone

> On Aug 17, 2015, at 10:45 PM, Khris Richardson <notifications@github.com> wrote:
> 
> In salt/states/saltmod.py:
> 
> > @@ -371,6 +379,7 @@ def function(
> >                  ).format(fun, tgt, str(False))
> >          ret['result'] = None
> >          return ret
> > +    cmd_ret = __salt__['saltutil.cmd'](tgt, fun, **cmd_kw)
> @gtmanfred @cachedout this looks redundant in light of the try block below
> 
> —
> Reply to this email directly or view it on GitHub.
>